packages/utils/src/providers.ts

```diff
@@ -147,9 +149,22 @@ const getCeloProvider = (): providers.Provider => {
   return provider;
 };
 
+const isEip1559Supported = async (provider: any, logger: LogTypes.ILogger): Promise<boolean> => {
+  try {
+    await provider.send('eth_feeHistory', [1, 'latest', []]);
+    return true;
+  } catch (e) {
+    logger.warn(
+      'This RPC provider does not support the "eth_feeHistory" method: switching to legacy gas price',
+    );
```

Comment:
I think this log should be removed from the utility

Reply:
I kept it there but made it optional.